parsec/backend/invite.py

```diff
@@ -119,6 +128,59 @@ def evolve(self, **kwargs):
 
 Invitation = Union[UserInvitation, DeviceInvitation]
 
+def send_invite_email(invitation: Invitation) -> None:
+    # Temporary infos for testing
+    sender_email = "pytestmailtest@gmail.com"
+    receiver_email = "pytestmailtest@gmail.com"
+    password = "mailtest"
+    
+    # mail settings
+    message = MIMEMultipart("alternative")
+    message["Subject"] = "You have received an invitation on Parsec Cloud"
+    message["From"] = sender_email
+    message["To"] = receiver_email
+    
+    # These strings are obviously subject to changes, especially to cover
+    # multiple languages (at least english & french)
+    lines = {
+        'line1': "Hi,",
+        'line2': f"You have received an invitation from {invitation.greeter_user_id} "
+                + "to join their workspace on Parsec.\nDownload now via the following link : ",
+        'line3': "Once installed, open the next link with Parsec",
+        'line4': "Or if it doesn't work, copy and paste it in the concerned section",
+        'line5': f"Lastly, get in touch with {invitation.greeter_user_id} "
+                + "and follow the next steps on Parsec to become part of their workspace.",
+        'button1': "Download Parsec",
+        'button2': "Invite link",
+        'parsec_url': "https://parsec.cloud/get-parsec",
+        'invite_link': f"parsec://localhost:6888/corp?action=claim_user&token={invitation.token.hex}&no_ssl=true"
+        }
+    
+    # Plain-text version used as backup if the html doesn't work for the client
+    text = f"{lines['line1']}\n{lines['line2']}{lines['parsec_url']}\n"\
+        + f"{lines['line3']}\n{lines['invite_link']}\n{lines['line5']}"
+    # HTML version
+    html = importlib_resources.read_text(module, "invite_mail.tmpl.html")
+    for item in lines:
+        html = html.replace(item, lines[item])
```

Comment:
You can take advantage of `str.format()` to do templating with less pain:

in your template
```python
<div>
<p>{line1}</p>
<p>{line2}</p>
</div>
<style>
        img {{
            border: none;
            -ms-interpolation-mode: bicubic;
            max-width: 100%; 
        }}
</style>
```

in invite.py:
```python
html_template = importlib_resources.read_text(module, "invite_mail.tmpl.html")
html_template.format(line1=line1, line2=line2)
```

We could improve this even further by using a real templating library such as [jinja2](https://jinja.palletsprojects.com/en/2.11.x/) but I think it's a bit overkill for our current needs

@c4ffein Any comment on the line1/line2/line... way of providing data into the template ?